lisa/schema.py

```diff
@@ -565,6 +565,65 @@ def _call_requirement_method(self, method_name: str, capability: Any) -> Any:
         return value
 
 
+@dataclass_json()
+@dataclass()
+class ACCOptionSettings(FeatureSettings):
+    type: str = "ACC"
+    is_supported: Optional[bool] = None
```

Comment:
As we discussed, can we use unsupported_features? If so, it doesn't need to add this flag.

Reply:
The reason I used schema is because this will allow extending any other capabilities to ACC easily. Do you think we should stick to just using features?

Reply:
Let's try unsupported_features firstly. If there are other settings, we can add them later.